js/server/process-vs.js

```diff
@@ -63,11 +67,19 @@ function processUserEvents (users, eventsByUser) {
       users[event.valSifAddress] = validator;
       validator.commisionClaimed = validator.commisionClaimed || 0;
       if (event.amount > 0) {
+        const isRedelegation =
+          userEvents.length > 1 &&
+          userEvents.some(other => {
+            return other.amount === -event.amount;
+          });
+        const redelegatedTicket =
```

Comment:
This should work for most cases, though if the user redelegates and splits their delegation into 2 different validators, or redelegates and joins 2 into 1, we may have issues.

Let's keep this in for now but flag an action item to review if there are any of the above cases to consider.

cc @caseyturnerarrington @nikohosn